sdk/core/azure-core/src/main/java/com/azure/core/http/HttpPipelinePosition.java

```diff
@@ -6,8 +6,22 @@
 import com.azure.core.http.policy.HttpPipelinePolicy;
 import com.azure.core.http.policy.RetryPolicy;
 
+
 /**
- * Indicates the position in an {@link HttpPipeline} to place an {@link HttpPipelinePolicy}.
+ * Enum representing the position in an {@link HttpPipeline} to place an {@link HttpPipelinePolicy}.
+ *
+ * <p>This enum encapsulates the positions where an HTTP pipeline policy can be placed in the HTTP pipeline. The positions
+ * are before or after a {@link RetryPolicy}.</p>
+ *
+ * <p>Each position is represented by an enum constant. For example, you can use {@link #PER_CALL} to represent the position
+ * before a RetryPolicy, and {@link #PER_RETRY} to represent the position after a RetryPolicy.</p>
+ *
+ * <p>This enum is useful when you want to add an HTTP pipeline policy to the HTTP pipeline and need to specify the position
+ * of the policy.</p>
+ *
+ * <p>Note: The positions are defined based on the RetryPolicy. The PER_CALL position means that the policy will only be
+ * invoked once per pipeline invocation (service call), and the PER_RETRY position means that the policy will be invoked
+ * every time a request is sent (including retries).</p>
```

Comment:
```suggestion
 * <p>This enum encapsulates the positions where an HTTP pipeline policy can be placed in the HTTP pipeline. The positions
 * are before or after a {@link RetryPolicy}.</p>
 *
 * <p>Each position is represented by an enum constant. Use {@link #PER_CALL} to represent the position
 * before a RetryPolicy, and {@link #PER_RETRY} to represent the position after a RetryPolicy.</p>
 *
 * <p>The PER_CALL position means that the policy will only be
 * invoked once per pipeline invocation (service call), and the PER_RETRY position means that the policy will be invoked
 * every time a request is sent (including retries).</p>
```

I think this might be clearer?